img/u/unimelb.edu.au.svg

```diff
@@ -0,0 +1 @@
-.011.041.026.116.026.116.044.013.725-.346.725-.346l.028-.014-.053-.1c-.101.013-.142.005-.166-.037l-.023-.092-.085-.506c.011.018.903-.451.913-.455.001.026.381.381.381.381l.058.067.004.01c0 .01-.01.024-.055.055l-.003.015c0 .041.03.098.03.098.044.013.745-.356.745-.356.013-.043-.026-.116-.026-.116-.082-.002-.1-.001-.179-.076zm-1.33-.38c-.083.013-.516.235-.611.285l-.221-1.192.027.026c.139.181.802.841.829.868l-.024.013z"/><g fill="#FFF"><path d="M41.894 35.093l-1.143-.41.594 1.112-1.08.763 1.218.273-.092 1.235 1.007-.838.93.838-.062-1.295 1.28-.108-1.065-.821.608-1.098-1.25.366-.47-1.189-.475 1.172zM49.222 36.982l-1.143-.38.593 1.096-1.082.748 1.251.273-.123 1.251.989-.841.96.841-.077-1.295 1.282-.109-1.05-.806.593-1.113-1.25.334-.457-1.156-.486 1.157zM41.011 43.29l.454 1.172 1.252-.35-.597 1.083 1.054.822-1.28.107.061 1.31-.929-.854-1.006.854.105-1.249-1.217-.275 1.067-.745-.58-1.082 1.141.379zM49.678 43.838l.367 1.082h1.126l-.913.701.35 1.113-.93-.67-.927.67.332-1.113-.912-.685 1.097-.016zM49.176 52.599l-1.144-.395.58 1.096-1.082.746 1.234.275-.091 1.25.991-.855.945.869-.091-1.28 1.295-.153-1.051-.822.595-1.082-1.235.351-.457-1.188-.489 1.188z"/></g><g fill="#FFF"><path d="M46.976 29.4c-.365-.003-.94.511-1.126.707.746-.258 1.026-.118 1.197-.342.01-.118.035-.294-.071-.365zM47.442 29.6c-.089.473.234.935.606 1.158l.068-.358c-.053-.336-.407-.646-.674-.8zM47.236 30.495c-.04.132-.157.301-.124.447.052.264.469.364.597.757.226-.363-.075-1.099-.473-1.204zM46.939 31.275l-.183.322c-.304.464.254.829.248 1.123.169-.181.358-.664.253-.998a.788.788 0 00-.318-.447zM45.816 31.346c-.201.079-.692.401-.942.366.123.156.664.124.857.148.337.02.566-.362.735-.63l-.65.116zM46.349 32.159c-.664.163-.244.943-.806 1.394.386-.07.554-.075.797-.537.11-.223.161-.629.009-.857zM41.178 30.682c-.127.507-.009.621.232.933-.117-.713.72-1.01.338-1.576-.268.069-.502.403-.57.643zM43.996 32.646c.025.158.657.12.949-.008.147-.032.641-.222.693-.4-1.145-.261-.737.164-1.642.408zM45.346 30.989c.052.15.668.002.935-.176.136-.059.587-.332.609-.516-1.174-.054-.699.291-1.544.692zM41.548 29.465c-.167-.077-.636.162-.757.249-.291.133-.742.58-.64.704.784-.512.829.074 1.397-.953z"/><g><path d="M43.124 27.467c.397.111.577.347.841.385.31.013.42-.079.479-.192-.125-.392-.916-.351-1.32-.193zM44.959 29.226c-.031-.413.132-.66.077-.922-.092-.297-.214-.37-.341-.386-.326.251-.022.979.264 1.308zM43.068 27.802c-.331-.117-.905.121-1.093.283.425-.042.693.431 1.199.224l.325-.171a.786.786 0 00-.431-.336zM43.722 28.633c-.106.333.149.901.319 1.084-.055-.425.409-.706.186-1.207l-.184-.32a.796.796 0 00-.321.443zM45.38 32.984l-.368.021c-.542.078-.531.621-.918.803.244.046.86-.045 1.084-.314a.78.78 0 00.202-.51zM41.657 28.511c-.494.177-.521.341-.646.714.527-.494 1.24.03 1.494-.602-.205-.184-.613-.189-.848-.112zM42.802 29.473c.044.522.195.592.523.807-.339-.635.352-1.191-.196-1.597-.232.15-.343.544-.327.79z"/></g><path d="M42.192 29.425c-.027.714-.015 1.262.29 1.603.194-.578.204-.839.048-1.189-.056-.154-.194-.422-.338-.414z"/></g></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 116.2 116.2"><path fill="#094183" d="M0 0h116.2v116.2H0z"/><path fill="#FFF" d="m13 77.3-.2.9v.1l.2-.1.1-.3c.2-.1.3-.2.8-.2h1.2V83l-.2.4h-.3v.1l.1.1h2.1c0-.1 0-.2-.2-.2h-.3c-.2 0-.3-.2-.3-.4v-5.3h1c.7 0 .9.2.9.4l.1.3s.2 0 .2-.2v-.9l-.1-.2H18l-.7.1h-3.8l-.3-.1H13l-.1.2zm11.2-.1h-1.1l-.2.1.2.1h.2c.3.1.3.2.3.5v2h-3.5v-2c0-.3.1-.4.3-.4h.2l.1-.2-.2-.1h-1.8l-.1.1.1.1h.3c.2.1.3.2.3.5V83l-.2.4h-.3v.1l.1.1h2v-.2h-.4c-.2 0-.3-.2-.3-.4v-2.8h3.5V83l-.3.4a26.4 26.4 0 0 1-.3.1l.2.1h2l-.1-.2h-.4c-.2 0-.2-.2-.3-.4v-5.1c0-.3 0-.4.3-.4h.2v-.2l-.1-.1h-.7zm5.4-.1-.4.1h-2.9l-.2.1.1.1h.3c.2.1.3.2.3.5V83c0 .2 0 .3-.2.4h-.2l-.2.1.2.1h2.9c.3 0 .5 0 .5-.2l.1-.8v-.2c-.2 0-.2.1-.2.2 0 .3-.2.4-.3.5l-.8.1c-.8 0-1-.1-1-.6v-2.2a117 117 0 0 0 1.6.3 26.1 26.1 0 0 1 .2.4l.1-.2v-.5l.1-.5v-.2l-.3.2h-1.6v-2.2h1c.6 0 .6.1.6.2a31.6 31.6 0 0 0 .2.5l.1-.2v-.5l.1-.3V77h-.1zm9.4.1h-1l-.2.1v.1h.4c.2.1.3.2.3.5v2.8c0 1-.1 1.6-.5 2-.3.4-.8.6-1.2.6-.5 0-.9 0-1.3-.4-.3-.3-.6-.8-.6-2v-3c0-.3.1-.4.3-.4a116.5 116.5 0 0 0 .3-.2l-.2-.1h-1.8l-.2.1.2.1h.2c.3.1.3.2.4.5v3c0 1.4.4 2 .7 2.2.6.6 1.4.7 1.8.7.4 0 1.1-.1 1.8-.7s.7-1.7.7-2.5v-2.7c0-.3.1-.4.3-.4h.2l.1-.2-.2-.1H39zm2.3.4v4.7c0 .8 0 1-.2 1a128.1 128.1 0 0 0-.4.2l.2.1h1.9c0-.1 0-.2-.2-.2h-.4c-.1-.1-.2-.3-.2-1l-.1-3.8 2 2.3a97 97 0 0 0 2.9 2.8l.1-.7v-4.9c0-.5 0-.6.3-.6a102.8 102.8 0 0 0 .3-.2l-.2-.1h-1.6l-.2.1.1.1h.4c.2.1.3.2.3.8v4l-.5-.6-1.3-1.4-2.7-3-.3-.1c-.2 0-.2.2-.2.5zm8.6-.4h-1l-.2.1.2.1h.2c.1 0 .2.2.2.5V83c0 .2 0 .3-.2.4H49l-.2.1.2.1h2l-.1-.2h-.4c-.2 0-.3-.2-.3-.4v-5.1c0-.3 0-.4.3-.4a113 113 0 0 0 .3-.2l-.2-.1h-.7zm7.5 0h-1l-.2.1.1.1h.3l.1.2v.3l-1.8 4.5a816980.2 816980.2 0 0 0-1.8-4.8l.1-.1h.3l.2-.2-.3-.1h-1.8l-.3.1.2.1h.3c.2.1.3.2.5.7l2 5.1.4.6c.1 0 .2-.1.3-.4l.1-.3 1-2.4.6-1.2.5-1.2.4-.8h.4l.1-.2-.2-.1h-.5zm4.6-.1-.4.1h-2.9l-.2.1.1.1h.3c.2.1.3.2.3.5V83c0 .2 0 .3-.2.4h-.2l-.2.1.2.1h2.9c.3 0 .5 0 .5-.2l.1-.8v-.2c-.2 0-.2 0-.2.2 0 .3-.2.4-.3.5l-.8.1c-.8 0-1-.1-1-.6v-2.2a117.3 117.3 0 0 0 1.6.3v.3l.2.1.1-.2v-.5a557 557 0 0 1 0-.7l-.2.2h-1.6v-2.2h1c.5 0 .6.1.6.2a165.4 165.4 0 0 0 .3.3v-.5l.1-.4V77H62zm3 3.4v-2.8l.5-.1c.9 0 1.4.6 1.4 1.6 0 .6-.2 1-.5 1.2a1 1 0 0 1-.6.2l-.8-.1zm-.1-3.3h-1.3l-.2.1.2.1h.2c.3.1.3.2.4.5V83l-.3.4h-.2l-.1.1.2.1h2c0-.1 0-.2-.2-.2h-.3c-.2 0-.3-.2-.3-.4v-2.1a179.4 179.4 0 0 1 1 .1l.3.4.7.8v.2c.5.5.8.9 1.1 1 .2.2.5.2 1 .2h.8l-.1-.2h-.4c-.2 0-.5-.2-1-.6a29 29 0 0 1-1.5-1.9l-.1-.2c.6-.6 1-1.3 1-2 0-.5-.4-1-.7-1.1-.4-.4-1-.4-1.5-.4h-.7zm5.4 1.4c0 .6.2 1.1 1 1.9l.5.3c.7.6.9 1 .9 1.5s-.4 1-1.1 1c-.3 0-1 0-1.2-.8v-.3l-.2-.1v.2H70v.8l.1.4c.4.2.7.3 1.2.3s1-.2 1.3-.4c.5-.4.7-1 .7-1.4 0-.7-.2-1.2-1.2-2l-.3-.2c-.8-.7-1-1-1-1.4 0-.5.4-1 1-1 .7 0 1 .4 1 .5l.1.4.1.2.1-.3v-1h-.3L72 77c-1.1 0-1.8.6-1.8 1.5zm5.6-1.4h-1l-.2.1.1.1h.3s.2.2.2.5V83c0 .2 0 .3-.2.4h-.2l-.2.1.2.1H77c0-.1 0-.2-.2-.2h-.3c-.2 0-.3-.2-.3-.4v-5.1c0-.3 0-.4.2-.4h.2l.2-.2-.2-.1h-.7zm1.9.1-.2.9v.1l.2-.1.1-.3c.2-.1.3-.2.8-.2H80V83c0 .2 0 .3-.2.4h-.3v.1l.1.1h2.1c0-.1 0-.2-.2-.2h-.3c-.2 0-.3-.2-.3-.4v-5.3h1c.7 0 .9.2 1 .4v.3s.2 0 .2-.2v-.9l-.1-.2h-.1l-.7.1h-3.9L78 77h-.1l-.2.2zm10.8-.1H88l-.2.1.1.1.1.2-.1.4-1.4 2.4a238.2 238.2 0 0 1-1.5-3h.2c0-.2 0-.2-.2-.2h-1.5l-.2.1.2.1s.2 0 .3.2a58.2 58.2 0 0 1 2 3l.2.8V83c0 .1 0 .3-.2.4h-.3l-.1.1.2.1h2v-.2H87l-.2-.4v-2.4a26.7 26.7 0 0 1 2-3l.4-.2.2-.1-.2-.1h-.6zm4.5 3c0-1.7 1-2.7 2.4-2.7 1.2 0 2.5 1 2.5 3 0 2.7-1.9 2.8-2.3 2.8-1.5 0-2.6-1.3-2.6-3.1zm-.9.2c0 1.6 1 3.4 3.3 3.4 2 0 3.4-1.5 3.4-3.5S97.6 77 95.6 77c-2.5 0-3.4 2-3.4 3.3zm11.1-3.2h-3.5l-.2.1.1.1h.3c.2.1.3.2.3.5V83l-.2.4h-.3l-.1.1.2.1h2l-.1-.2h-.4c-.2 0-.2-.2-.2-.4v-2.6h1.2c.2 0 .3.2.4.3v.3l.1.1.1-.2v-.5l.1-.5v-.1h-.2c0 .1-.1.2-.3.2h-1.4v-2.3h1.2c.3 0 .5.1.5.2l.1.3.1.2.2-.2v-1.1h-.1zM13.8 88.5H103v.5H13.8zm9 5.3-3.5 7.4a64284.7 64284.7 0 0 1-3.9-7.7s-.2 0-.2.4l-1.3 8.4c0 .5-.2 1-.5 1H13s-.2 0-.2.2l.2.1h2.7l.3-.1c0-.1 0-.2-.2-.2h-.6c-.2 0-.3-.2-.3-.4v-.6l.7-5.1.6 1.3.7 1.5 1.5 3c.2.4.4.6.5.6.1 0 .2-.1.5-.7l2.8-5.9.8 5.7v.5c-.2 0-.2 0-.2.2l.4.2a69.8 69.8 0 0 0 3-.1c0-.2-.2-.2-.3-.2h-.8c-.3-.2-.5-.6-.6-1.4l-1.2-8c0-.2 0-.4-.2-.4l-.3.3zm10.1-.1-.6.1h-5.1l-.3.1c0 .2.1.2.2.2h.5c.4.1.5.3.5.7v7.8c0 .4-.1.6-.4.7h-.4c-.1 0-.3 0-.3.2l.3.1h5.2c.5 0 .7 0 .8-.3l.1-1.4v-.3c-.2 0-.2 0-.3.2 0 .6-.2.8-.5 1l-1.2.1c-1.4 0-1.5-.2-1.5-1v-3.2h1.8c.4.1.7.3.7.6v.4l.2.2c.2 0 .2-.2.2-.3v-.9l.1-.8v-.1l-.1-.2-.2.2-.6.2a43.2 43.2 0 0 1-2.1 0v-3.4h1.8c.7 0 .8.2.9.4v.5s0 .2.2.2l.2-.2v-.8l.1-.7v-.3h-.2zm5 .1h-2.4l-.4.1c0 .2.2.2.2.2h.5c.4.1.5.3.5.7v7.8c-.1.4-.2.6-.4.7h-.5s-.2 0-.2.2l.3.1h5.4c.7 0 .7 0 .8-.3l.2-1.5-.1-.3c-.2 0-.2.2-.2.3l-.3.7c-.3.4-.8.4-1.5.4-1 0-1.3-.1-1.5-.3-.3-.2-.3-1.1-.3-2.8v-5c0-.4.2-.6.5-.7h.5c.1 0 .2 0 .2-.2l-.3-.1h-1zm7.9 0h-2.7l-.3.1c0 .2.2.2.3.2h.5c.4.1.5.3.5.7v7.8c0 .3-.1.6-.4.7h-.4c-.1 0-.3 0-.3.2l.3.1h3.5c2.4 0 3.5-1.4 3.5-2.8S49 98.4 48 98c.7-.6 1.3-1.2 1.3-2.2 0-.5-.2-2-2.8-2h-.7zm.5.6c1.1 0 1.5 1.2 1.5 2 0 .6-.1 1-.5 1.3-.2.2-.5.2-.9.2h-.6v-3.5h.5zm-.1 8.5c-.4-.2-.5-.4-.5-1.2v-3.1h.6c.5 0 .8 0 1.1.3a3 3 0 0 1 1.2 2.5c0 1.5-1.2 1.6-1.5 1.6l-1-.1zm5.4-4.2a5 5 0 0 0 5.2 5.1c3.2 0 5.4-2.2 5.4-5.3 0-3-2-4.9-5.2-4.9-3.7 0-5.4 2.7-5.4 5.1zm1.8-.4c0-3.5 2-4 3.2-4 2.2 0 3.7 2 3.7 4.7 0 3.7-2 4.1-3 4.1-2.2 0-3.9-2-3.9-4.8zm18.7-4.5h-.4a697.5 697.5 0 0 0-1.7.1c0 .2.1.2.2.2h.5c.4.1.5.3.5.7v4.4c0 1.4 0 2.5-.7 3.2-.6.6-1.5.7-1.8.7-.4 0-1-.1-1.6-.6-.5-.4-1-1-1-3.1v-4.6c0-.4.2-.6.5-.7h.4s.2 0 .2-.2l-.3-.1h-3.5l-.3.1c0 .2.2.2.3.2h.5c.4.1.5.3.5.7v4.7c0 1.6.4 2.7 1.2 3.4 1 1 2.3 1 2.9 1a4 4 0 0 0 2.7-1c1-1 1.2-2.5 1.2-4v-4.1c0-.4.2-.6.5-.7h.4c.1 0 .2 0 .2-.2l-.3-.1h-1zm5.6 0H75l-.3.1c0 .2.1.2.2.2h.5c.4.1.5.3.6.7v7.8c-.1.3-.2.6-.4.7H75s-.2 0-.2.2l.3.1h3.4l.3-.1s0-.2-.2-.2H78c-.3 0-.4-.3-.5-.7v-3H79l.6 1 .6.9c.8 1 1.3 1.6 1.8 1.9.4.2.7.2 1.4.2h1.3l.2-.1s0-.2-.2-.2h-.3c-.2 0-.7-.2-1.4-.9l-2.4-2.8-.4-.5c1.2-1 1.7-1.9 1.7-3a2 2 0 0 0-1-1.7c-.6-.5-1.5-.6-2.3-.6h-.9zm0 5-.1-.2v-4l.7-.1c.8 0 2 .5 2 2.3 0 1.2-.5 1.7-.8 2l-.9.1-1-.1zm8.8-4.8-.1 7.7c0 1.1-.1 1.5-.4 1.6h-.6s-.2 0-.2.2l.3.1h3l.3-.1c0-.2-.2-.2-.2-.2a2 2 0 0 1-.7 0c-.3-.2-.4-.6-.4-1.5l-.2-5.3.3.2 2.8 3 1.7 1.7 2 2c.2.2.4.4.6.4.1 0 .2-.2.2-.5l.1-8.1c0-.7.1-1 .4-1a497.5 497.5 0 0 1 .7-.3l-.3-.1H93l-.4.1c0 .2.1.2.2.2h.7c.3.2.4.4.4 1.2l.1 5.8-1-1-1.8-2-4-4.1-.1-.1-.4-.3c-.2 0-.2.3-.2.5zm16.2-.3-.6.1H97l-.3.1c0 .2.1.2.2.2h.5c.4.1.5.3.5.7v7.8c0 .3-.1.6-.4.7h-.4c-.1 0-.3 0-.3.2l.3.1h5.2c.5 0 .7 0 .8-.3l.1-1.4-.1-.3-.2.2c0 .6-.2.8-.5 1l-1.2.1c-1.4 0-1.5-.2-1.5-1v-3.2h1.8c.4.1.7.3.7.6v.4l.2.2c.2 0 .2-.2.2-.3v-.9l.1-.8v-.1l-.1-.2-.2.2-.6.2a43.1 43.1 0 0 1-2.1 0v-3.4h1.8c.7 0 .8.2.9.4v.5s0 .2.2.2l.2-.2v-.8l.1-.7v-.3h-.2zM58 70a22.4 22.4 0 0 0-7.4-4.4h-.2L49 65l-.3-.1-.2-.1A19.8 19.8 0 0 1 34.6 45a37.8 37.8 0 0 1 .6-6.9l.8-4.4.1-.8v-.3l.1-2.1c0-3-.8-5-2.5-6.3l-.2-.2 3.7-5c-.4-.5-.6-.9-.6-1.3v-.5l.5.1 1 .4h.2l3.7-4.9.2.3c.7 1.2 2 1.8 3.8 1.8l2.8-.3 3.7-.6 3.1-.3h.2l2.4-.2a36.1 36.1 0 0 1 7.8.8l3.3.6h1.5c1.5 0 2.5-.3 3.2-1l.5-.4.3-.5 3.5 4.5h.1l2.3-.9.7-.2 3.1-1 .7-.1-.3.6-.6 1.3-.4.6.6-.3 1-.3-.5.8a7 7 0 0 1-2 2.5l-.1.1v.2h.1l.1.7v.6l-.6.7-.1.1.9 1.3-.2.1c-1.8 1.3-2.6 3.4-2.6 6.3v.6l.1 2.1.5 2.9.4 2.9.4 3.1.1 3v.3a19.8 19.8 0 0 1-14.2 19.4 27 27 0 0 0-9.2 4.9h-.1l-.1.2-.2.2-.2-.2z"/><path fill="#094183" d="M82.3 23.7h-.1.1zm0 0zm2.1-6.4.8-1.3.4-1-1.1.2-3.2 1-.6.2-2.3.8-3.6-4.6-.5.7-.4.5c-.7.6-1.7.9-3 .9h-1.6l-3.2-.6-2.7-.4a36.3 36.3 0 0 0-7.8-.3l-3.1.4-3.8.5-2.7.3c-1.8 0-3-.5-3.6-1.6l-.4-.6-3.8 5c-.4 0-.7-.2-1-.3l-1-.3.2 1c0 .4.2.8.5 1.3L33.2 24l.3.4c1.8 1.2 2.5 3 2.5 6l-.1 2.4-.1.8-.8 4.5-.4 2.8a38 38 0 0 0-.2 5.2 20 20 0 0 0 14.2 19l.4.1 1.3.5.3.1a22 22 0 0 1 7.3 4.4l.3.3.4-.4.2-.2c2.6-2.3 5.6-3.5 9-4.8a20 20 0 0 0 14.4-20v-2.9l-.4-3.2-.5-2.9-.4-2.8a18 18 0 0 1-.2-2.6v-.1c0-2.8.9-4.8 2.6-6.1l.3-.3-.9-1.3c.3-.2.4-.5.6-.8.1-.3.2-1 0-1.4h-.1c.9-.7 1.8-2 2-2.6l.8-1.4-1.5.6zm-45.4.4 3-4c1 1.6 2.8 1.7 4.5 1.8l1.1-.2 5-.7 3.2-.4c3.2-.4 6.4.1 9.4.5l4.3.8c1.9 0 3.8 0 5.1-1.3l.2-.3 2.8 3.6-.3.2c-.3 0-.7.2-1 .4H76l-1.5-1.8a9 9 0 0 1-5 .8l-2.6-.4-3.1-.5-3.4-.4c-2-.2-4 0-6 .1l-2.8.4-4.6.7c-1.7.2-3.5 0-4.9-.8L40.8 18h-.6l-1.2-.3zm35 8.6c0 .5-.5.6-.7 1l-.2.1.6-.3c.3-.2 1.5-.4 1.5-.2 0 .3 0 .5-.4.7s-1.5.8-2.4.6c-.5 0-1 .2-1.4.3l-2.4.7a5 5 0 0 1-2.7 0c-1.4-.2-2.7-.5-4-.4-.2 0-.5.1-.6.3h.6c0 .2 0 1.2-.5 1.6l-.2-.5c0-.2-.6-2.7.5-3.5.7-.2 1.5.2 1.7.3l3.6 1 1.5-.2 3.4-.5c.7.1.8-.7 1.3-.7.3 0 .6-.3.8-.4h.2zM45.8 28c.1-.2.6-.3 1-.2.5 0 1.2.3 1.8.7 1.2.4 2.1 1.2 3.3 1.6l-.1-.3a11 11 0 0 1 2.3-1.5c.6-.6 1.3-1.2 2.2-1l.2-.1v.3c0 .4-.3.9-.6 1.5l-1 .5-2.9 1.6c-.7.6-1 0-1.8-.5L47.7 29c0-.1.2 0 .1-.2-.4 0-.7.2-1 .3-.2.1-.6.2-.8 0h.2c.2-.2.3-.5.7-.6-.3-.3-.8-.4-1.2-.4l.1-.1zm31.8.2c0 .2-.4.7-.9.8a2 2 0 0 1-1.6-.2c0 .4.8.6 1.3.5-.3.5-1.2.8-2 .7-.7 0-1.2-.2-2-.6 0 .4.9.8 1.5.9-1.2.6-2.6.1-2.8 0-.3 0-.8-.2-1-.5 0 .4.4.7.8.8-.8.3-1.8 0-2.2-.8l3.7-1c1.3.2 2.5-.3 3-1 .2-.1.3-.4.3-.5h.4c.1 0 1.4.4 1.5.9zm-11 1.9c.7 0 1.5 0 1.8-.2.2.3-.1.7-.4.6.2.4.5.8.4 1.2l-1.2-.4c.2.3.5 1 .4 1.4l-1.4-.5c.1.6.5 1.4 0 1.9-.4-.3-.7-.8-1-1.2 0 .5.1 1.3-.2 1.8-.3-.3-1.3-.7-1.2-1-.1 0-.4 1-.7 1.3-.4-.5-.7-1.2-1.1-1.6.1-.1.3 0 .2-.3-.1-.3-.6-.5-1.1-.7 0-.2.1-.3.3-.5l.8-1.5.2-1 1.6.3c.7 0 1.7.3 2.6.4zm-11.3-.2-.4.5c-.2.5 0 1.2.5 1.6l.8.4c-.7.1-1.4 1.1-1.4 1.6-.3-.2-.4-.7-.3-1.1-.2.2-.6.4-.9.4 0-.2 0-.8.3-1-.1-.2-.7-.1-1 0 0-.3 0-.5.4-.8 0-.2-.3 0-.8 0l1.3-.7 1.5-1zm.3 14 1.2 1.9 2.3 3.7c.4.5 2 3.4 3.1 4-.8.9-.4 2-1.8 2.2-.8.1-3 .8-3.8-.5s-1.1-3.3-.9-5a7 7 0 0 0 2.2 2.5c-1.8-2-1.5-2.4-2.4-3.5-.4-.4-.4-1.1-.5-1.6v-2.8c0-.5 0-1-.2-1.5l-.3-.9V42l1.1 2zM61 26.4c-.8.5-1 1-2.2 1.6-.7.4-1.4.7-1.8.5-.5-.3-.3-1-.3-1.2V27c.3-.2.8 0 1-.5l.1-1c.2-.4.6-.5.7-.9-.6.4-1 1-1.7 1-.3-.1-.1-.4-.2-.6-.2 0-.2-.1-.2-.2h.1v-.1h-.2v-.4l-.4-.1c0-.4.3-.7.3-1 0 0 .3 0 .2.1v.2c0-.1.6-.4.3-.5H56c0-.4.1-.6.3-.9.3.3.8.6 1 1 0 .2.2.4.4.4.4-.2.6.3 1 .2 0 0 0-.2.2-.2l.2.1c0 .2 0 .3-.2.4a1 1 0 0 1-.5.2c.4.2.6-.1 1-.2 0 0 .2 0 .3.2.1.7-.2 1.8.7 2.2h.5zm14.5 27.1c-.6 0-1 0-1.3.6l-.7 1.6c-.3.5-.8.9-1.5 1.1-.8.2-2 .3-2-.2.1-1 1.4-1.1 2-1.4 1-.5 1.1-1.4 1.2-2.3l-4.4-3.3s2.5 2.6 3.9 3.5c0 .8-.2 1.2-.9 1.6l-1.4.7c-1 .5-.7 1.3-2 1.8-.6.3-2 .4-1.6-.3.5-.8 0-.8 0-.8l-1.6.4c-.3 0-.6-.2-.3-.4l.3-.3.4-.4c.2-.2.3-.6.1-.7l-.6.1c-.5.7-.6 1.7-2.2 1.4 0 0-1.9-.4-.3-2.6.4-.5-.8-1.2-1.5-2-1.3-1.7-5.1-7.9-5.1-7.9L54.8 42c-.4-.5-.5-1.7.3-1.6.4 0 .6.3 1 0 .3 0 .3.4 0 .7-.2.1-.2.5 0 .7.4.2.5-.2.9 0 .3 0 .6 1 1.2 1.1h.4l-.2-1 .8.6c.5.2.7.3.6-.1l-.2-.7c0-.4.3-.6.5-.6.1 0 .3.1.3.3.3.5.2.8.9 1 1 2 3.5 4.8 4.5 5.6l1 .5-.8-.7c-2.7-2.5-4.4-5-4.4-5.4.8-.3 1-1 1.1-1 .2 0-.2.7-.1 1 0 .2.2.4.7.4-.9.9 1.3 1.6 3 1.6.5.3 1.6 1 2.8 1.3 0 0 .1.5.5.6l2.1 2.7c1.3 1.3 2.7 3.5 3.8 4.6zM66 52.1c-2.7-2.9-5.3-7.1-5.6-7.3l3.8 5.8L66 53c.1-.2.3-.5 0-.8zm-4.8-25.2c-1.1 1 .2 3.8-.6 4.9l-.2.4-1.5-.3-1 .1-1.5.1.2-.4c-.5 0-1 0-1.2-.4a1 1 0 0 1-.1-.9c.1-.4.5-.7.7-1.1l.4-1c0 .1.2.4.7.6.4 0 1-.1 1.8-.5-.4.6-1 1-2 1.1.4.1 1 0 1.8-.4a8 8 0 0 0 1.8-1.9l.7-.3zm-1.3 3.8c-.2.3-.4.6-1.1.6-1 0-1.2-.5-1.2-.5 0 .3.3.9 1.2.9 1.1 0 1.1-1 1.1-1zM82.5 24a7 7 0 0 0-2.3 4c-.3 2-.4 4 0 5.8l.9 5.7.2 2.5.1 3c.2 6.7-3 15.7-14.1 19.4l-2.7 1a23 23 0 0 0-6.4 3.8 28 28 0 0 0-9-4.9 19.3 19.3 0 0 1-14-17.7c0-2 0-4.1.2-6.1l.5-3 .7-4.1a15 15 0 0 0-.5-7 7 7 0 0 0-1.9-2.6 582 582 0 0 1 3-3.8l.5 1 .3.5-1.6 2.1c2 2.3 2 5.5 2 8.5L38 34l-.9 5.3-.3 3-.1 2.8c-.4 6.8 4.8 15.5 14.3 18.3 2.5 1 4.9 2.2 7.1 3.8a31 31 0 0 1 7.1-3.7 18.6 18.6 0 0 0 14.5-17.2c.1-2 0-4-.3-6l-.4-2.9-.6-3.5c-.3-2-.3-4.3 0-6.2.7-.2 1.3-.8 1.7-2v-.2a9 9 0 0 0 2-1.9l.3.5zm2.1-8.3c-.8 1.7-1.2 2.2-2.4 2.6L77.4 20 75 21.2c-2-1.2-5.7-.3-6.6 0a19 19 0 0 0-4.2 2.3l-.6-.5s.6.4.5 1c.1 0 .8-.4 1.2 0 .4.3 0 .8-.2 1.1.5-.3 1-.6 1.5-.3.5.4-.1 1-.3 1.3.4 0 .8-.3 1.1 0 .3.5-.1.7-.5 1 .4 0 .4.2 1.5.3l-1.4.1-1-.3c-1.4-.4-2.1-.8-2.7-1-.5-.1-1-.2-1.5 0h-.3c1.7-.6 1.2-2.6 2-3.9.8-1.6 2.6-2 4.1-2.2l6.1-.8c1.7-.3 3.8-1 4.2-1.2l3.7-1.4 3.1-1zm.1 2c-.2.7-1.1 2.2-2 2.7-1.4.7-3.2 1-4.6 1.2-1.1.2-2 0-2.4-.2l-.6-.2a35 35 0 0 1 7.4-2.7l2.2-.7zM58.2 20c-.6.1-1.3.2-1.7 1-.2-.2-.1-.5 0-.8.3-.2.8-.4 1.7-.2zm24.5 2-.7 1c-.9.6-5 1.2-6.5.7 1.7-.7 1.9-1.2 2-1.6 1.2 0 3-.2 5-1.1.3 0 .4.7.2 1zM74 21c.5.2 1.2.5.9.6-2.6.4-4.3 1.2-8.7 2.9h-.5c0-.2 0-.5-.2-.7-.3-.2-.6-.3-1.1-.1a23 23 0 0 1 4-2.3c1.3-.4 2.5-.6 3.9-.6.8 0 1.3 0 1.7.2zm-13.2.2c0 .1 0 .5-.3.6-.5.2-.8-.3-1.2-.4-.8-.1-1.7 0-2.4-.2a2 2 0 0 1 1-.7c.4-.1 1.3-.3 2-.1.5.2.8.5.9.8zm.6.2-.1.3s-.2.2-.4.1c0-.2 0-.4.3-.5l.2.1zm15.9.6c0 .4-.6.8-1 1-.5.3-1.1.5-1.4.8-.3.3-2.8 1-4 1.3l-3.3.8c-.3-.2-.6-.2-.8-.2.3-.6.2-1-.4-1.1l5.9-2.2 3.3-.7c.6.2 1 .3 1.7.3zm-17.8-.2c.4 0 1 .8 1.3 1.1 0 .3-.1.6-.3.8l-.4.4v-.5a1 1 0 0 1-.5-.5c-.6 0-1.5 0-1.8-.2 0-.2 0-.3-.2-.4h1.3c.4 0 .9 0 1 .2l-.1.1h-.1c0 .2.3.2.5.2.1 0 .2-.2.1-.3-.1-.3-.4-.3-.7-.3l-2.1-.1-.9-.6c0-.2.1-.3.2-.2.7.2 2 .2 2.7.3zm2.3 0c0 .2 0 .2.2.3 0 .2-.2.4-.3.5H61l-.3-.1c0-.4.4-.6.7-.7h.3zm-17.5.4c1.7.3 1.6 1.8 2 2.4-1.4.1-2 .2-3.1 0-3.9-1-4.6-2.8-5.7-5.2 1.9 1.6 6 2.6 6.8 2.8zm37.6 1a8.8 8.8 0 0 1-3 2.4 3 3 0 0 1-2 .1c-.4 0-.9-.3-1.2-.5-.6-.5-.6-.9-.6-1 0-.2.2-.5.4-.4 2.4.6 5.7-.1 6.4-.6zm-20.4-.3h.2c.1.3-.2.4-.4.4s-.3 0-.3-.2l-.1-.3v-.1c.2.2.4.3.6.2zm-15 2c0 .2.3.7 1.1 1.3l-1.6.2h-2c-1.4-.2-3.2-1.9-3.6-2.5a10.2 10.2 0 0 0 6 1zm4.5 1.2c0 .2 0 .4.4.6l-1 .1c-1.4 0-2.9-.5-3.7-2A11.5 11.5 0 0 1 51 26zm26 0c0 .2-.6.7-1.2.8 0 0-.1-.5-.8-.5-.4 0-.4.1-.6.2V26l-.2-.1a1 1 0 0 0-.6.2c-.2.1-.4 0-.6.2-.3 0-.5.3-.8.6h-.3l-2.6.3c-.5 0-1.1 0-2-.2 0 0 .7-.3.4-1l4.7-1c1.4-.3 2-.6 2.5-.9-.1.3.2.9.7 1.3.4.3 1.1.6 1.4.6zm2.8-.7a2.4 2.4 0 0 1-3.6 1.6c.8-.2 1.1-1 1.1-1 1 .2 1.8 0 2.5-.6zM50.2 27c-.4.1-.3.4-.2.6.1.2.4.3.4.3l-.5.5-1.2-.6a4 4 0 0 0-2.4-.7l-1 .3c-.5-.1-1.3-.6-1.5-.8a10 10 0 0 0 4-.2c.7.3 1.3.6 2.4.6zm11.3 6c-.3 0-.1.2-.7.6l-.8.1c0-.2.2-.3 0-.7-.2-.5-.5-.6-.7-.6l.6.7c0 .2 0 .4-.2.6l-.6.3c.2-.5-.3-1.3-.7-1.4.4.4.7.8.5 1.5-.1.3-1.2 1-1.5.8-.5-.5-.1-.8-.3-1.4 0 0 0-.2-.2-.1.2.5 0 .7 0 1.2-.4.1-.5.2-.8.1-.1-.1-.2-.3-.1-.7 0-.4.3-1 .7-1.3 0 0-.2 0-.3.2-.5.4-.7 1-.7 1.4 0 .2 0 .3.2.4-.1 0-.6 0-.8-.3-.2-.5.2-1 .5-1.4.2-.4.6-.5.6-.5.7-.4 1.6-.2 2.3-.3.6-.1 1.2 0 1.7.1s.9.4 1.3.7zm10 11.4c0 .4-.5.7-1 .7h-.1c0-.4 0-1.1.4-1.2.4 0 .6.1.6.5zm-6.7-4.7c1.2 1.5 2.3 2.2 2.6 2.4 0 0-1.7-1.5-2-2.1 2.8 2.1 5 3 6 3.7-.4 0-1-.1-1.1.2-.3.3-.1.8-.2 1.2-.1.4-.4.4-.5.4-.2-1.3-1.2-1.5-1.2-1.5.3.3.6.6.7 1-1 0-2.8-1.2-2.8-1.2-1 0-2.1 0-2.7-.5-.7-.4.7-.5.1-1-.2-.2-.5 0-.8-.3-.1-.2.4-.7.2-1.1-.1-.3-.4-.3-.7-.1-.2.2-.4 1.2-1.1 1-.4-.1-.3-.8-.8-1.2a1 1 0 0 0-1 0c-.3.4-.2.8-.2.8-.9-1.7 0-5.4 0-5.4-1 2.4-1 4.4-.2 5.7a2 2 0 0 1-.7-.7c-.2-.4-.5-1-.4-1.7-.6 1 .5 2.8 0 2.8-.3 0-.3-.7-.7-1-.5-.2-.6.2-.8 0-.2-.1.7-.8 0-1.3-.5-.2-.7.2-1.1 0-.4-.2-.9 0-.9 0-.1-1 1-4.7 1-4.7.6.3 1.5-.2 1.5-.2.4.9 1.4.1 1.8-.2.3-.3.4-.5.8-.7.4-.2.9.1 1.3 0l.5-.5a9 9 0 0 1 1.5 2.9c1 2.1 2 3.3 2 3.3zm-8.6-13.3c-1-.1-1.8.3-2.5 1.2-1 .3-1.5.7-2.3 1.4-.5-.1-1-.5-1.3-.6.2-.3.3-.5.6-.6-.2-.1-.6-.3-.6-.6.2-.5.8-.2 1.5-.5-.2-.1-.6-.4-.4-.8 0-.4.7-.3 1.2-.2-.2-.3-.3-.7 0-1h1c0-.4.2-.7.4-1-.3.1-.4.5-.5.7-1 0-1.4.4-1.2 1-.5 0-1 0-1.1.4-1.2-.7-2.9-1.2-4.1-1.3h-.4c-.2-.1-.3-.5-.3-.7-.2-.5-.4-1.2-1-1.5 1.2.2 4.2.8 6.1 1.5-1.4-1-5.2-1.5-7.5-2-1.7-.4-4-1.2-5.5-2.2-.8-.6-1.3-1.3-1.4-2 1 .4 2 .5 3.2.8l2.6.5 2.7.7c2.4.7 5.1.3 7.5 1 1.1.5 1 1.8 1.2 2.8l.3.3c0 1.3.6 1.9 1.8 2.7zM64 39.7c1.1 1.5 1.2 1.5 2.2 2.5a15 15 0 0 1-2.4-2.2 7 7 0 0 1-1.6-3.4s.6 1.7 1.8 3.1zM51.4 22c-1.4-.8-3.4-.7-4.6-1l-1.3-.4 1.4.3c1.5.2 3.2.3 4.5.8.7.3 1.1 1 1.3 1.8-.3-.6-.7-1.2-1.3-1.5z"/><path fill="#094183" d="M24.4 53.8h-.2v.3c.7 2.5 0 5-1.8 6.6l-.6.5.8-.1 2.1-.2c1.9.1 3.2.5 4.3 1.2a30027.6 30027.6 0 0 0-1.2 5.9c0 .3.2.7.4 1 .8.8 1.9 1 2.9 1.4 4.5 1 9.2.1 13.2-2.6l2.2-1.4 1-.6 1.1-.7A17.2 17.2 0 0 1 67 65l.5.3a3878.5 3878.5 0 0 0 4.6 3c4 2.5 9.1 3.2 13.4 2 .7-.3 2.2-.8 2.5-2.1l-1.2-6h.2c1.7-1 4.1-1.4 6.3-1l.8.2-.6-.5a6.4 6.4 0 0 1-1.9-6.7l.1-.2h-.3c-2.7 0-5.3.3-7.6 1.1-1 .4-2.8 1-3.3 2.6V64c-2.8 0-5-1.4-6.8-2.6a822 822 0 0 1-3.4-2.2 24 24 0 0 0-9.5-3.6c-5.6-.7-11.2.7-16.2 4L42 61.4c-2 1.3-4 2.5-6.7 2.6v-6c0-.9-.7-1.7-2-2.4a19.8 19.8 0 0 0-8.9-1.8z"/><g fill="#FFF"><path d="M32.7 60.3c.7-.2 1.4-.5 2-1V64c-1.6 0-3.3-.3-4.6-1.5-.2-.2-.4-.4-.4-.6l.1-.2c.4-.7 1.1-1 1.9-1.1l.6-.2.4-.1zm48.3-.8a6 6 0 0 0 2.5 1.2 6 6 0 0 1 2.2.9c.2.2.3.3.3.5v.2c-.5.8-1.5 1.2-2.2 1.4-.9.2-1.8.4-2.7.4v-4.6z"/><path d="m25 56-.1-1.6c2.7.1 5.7.4 8.4 1.9.7.4 1.4 1 1.4 1.8 0 1-1.1 1.3-2 1.6l-1.1.3c-1 .3-2 .6-2.4 1.6a9.3 9.3 0 0 0-5.7-1.3c1-1.2 1.6-2.7 1.6-4.4zm59.9 4.4h-.1l-1.2-.3c-.9-.3-1.7-.5-2.4-1.2a1 1 0 0 1-.1-.6v-.6c1-1.3 2.3-1.7 3.7-2.2h.4a24 24 0 0 1 5.8-.8l-.2 1.6c0 1.6.5 3.1 1.5 4.3-2.2-.1-4.2.3-5.7 1.2-.3-.8-1-1.1-1.7-1.4zm-43.2 2 3.9-2.5a22.2 22.2 0 0 1 23.8-.4l2.5 1.6 2.6 1.6c2.4 1.7 5.4 2.4 8.4 2 1.2-.3 2.5-.7 3.4-1.7l1 5.3c-.1.7-.9 1-1.5 1.3h-.1c-4.5 1.5-9.9.6-14.1-2.3l-3.9-2.5a18.4 18.4 0 0 0-20.6.4l-3.5 2.3A16.6 16.6 0 0 1 32 70c-1.2-.3-2.4-.5-3.4-1.4a1 1 0 0 1-.3-.6s1-4.6 1-5.2c1.4 1.4 3.2 1.7 5 1.8 2.5.1 4.7-.6 7.3-2.2z"/></g><path fill="#094183" d="m31.7 65.5-1.4-.2v.1l.1.2-.4 2.6v.2l.8.1v-.2l.1-1h.6c.8.2 1.2-.3 1.2-.7.1-.5-.1-1-1-1zm-.3 1.3-.5-.1.1-.7h.5l.5.2v.3c0 .3-.2.3-.6.3zm3-1c-.4 0-.8.1-1 .4-.3.3-.5.7-.5 1.2 0 1 .5 1.6 1.3 1.6.4 0 .8 0 1-.3.3-.3.5-.7.5-1.2s-.1-1-.4-1.3c-.2-.2-.6-.4-1-.4zm.4 2.5-.5.2c-.6 0-.8-.4-.8-1.1 0-.4.1-.7.3-.8.1-.2.3-.2.5-.2s.4 0 .5.2c.2.2.3.5.3.8 0 .4-.1.7-.3.9zm2.7-1.6c-.4 0-.7 0-.7-.2s0-.4.3-.4c.5-.1.8 0 .8.1v.1h.1l.3-.7v-.1l-.2.1h-1c-.8.2-1 .7-1 1.1.2.6.8.7 1.4.7.3 0 .7 0 .7.2v.3l-.5.3-.8-.1-.1-.2-.3.8h1.2c1-.2 1.3-.8 1.2-1.2-.2-.8-.9-.8-1.4-.8zm3.7.5-1-2 .6-.3h.3l-.3-.7-.2.2-1.8.8h-.3l.3.7.2-.2.6-.3a62.1 62.1 0 0 0 1 2.4l.8-.4c-.2 0-.2 0-.2-.2zm3.2-2.4-.1.2-1.2.8-.3-.7.5-.4h.3l-.3-.7h-.1v.1l-.2.1-.6.4-.3-.6 1-.6c.1-.1.1 0 .2 0 .1-.1-.3-.7-.3-.7l-.2.2-1.7 1.1v.1h.3l1.4 2.2v.3l1.8-1.1h.3l-.4-.7h-.1zm2.6-1.4h-.1l-.9-.5c.4-.3.4-.7.1-1-.3-.5-.8-.6-1.3-.3l-1.3.8v.1l.3.1 1.4 2.2v.2l.8-.4v-.1h-.3l-.5-1 .4-.2.9.5.1.1v.2l.8-.4v-.1l-.4-.2zm-1.6-.6-.5.3-.4-.6.5-.4c.3-.1.5-.1.6.1v.3s0 .2-.2.3zm7.4-2.6c-.1 0 0 .2 0 .2v.2l-.5.3h-.6a1 1 0 0 1-.5-.7v-1l.5-.3c.3 0 .6 0 .6.2h.1l.2-.7v-.1h-1.1c-.7.3-1.3 1-1 2 .3 1 1.1 1.4 1.9 1.1.5-.1.7-.4.8-.5l.1-.2h.2l-.7-.5zm3.2.1h-.1l-.6-.9c.4-.1.6-.5.6-1-.1-.5-.6-.7-1.2-.7l-1.5.2v.2l.2.1.4 2.6-.1.1v.1h.8v-.2l-.1-.1-.1-1h.4l.6.8.1.1v.1c0 .1.8 0 .8 0v-.1l-.2-.3zm-.8-1.4-.4.1-.6.1-.1-.7.6-.1c.3 0 .5 0 .5.3v.3zm3.4 1.2h-1.3v-.8h.7l.2.1h.1v-.7h-1v-.6h1.5v-.6h-2.2l-.1.2.2.1v2.6l-.2.1v.1h2.1l.2.1h.1V60l-.3.1zm2.7-1.3.6-.7h-.4l-.1-.2-.7-.2c-.8-.1-1.2.3-1.3.8 0 .6.6.8 1 1 .4.1.7.2.7.5l-.1.2c-.2.1-.4.2-.6.1-.3 0-.6-.1-.7-.3l-.1-.1-.5.7h.3l.2.1.7.3c.5 0 .9-.1 1-.3.3-.1.4-.3.4-.5.1-.8-.5-1-1-1.2-.3-.1-.6-.2-.6-.4l.1-.2.4-.1c.4 0 .5.2.6.3v.2h.1zm3 2.6-.3-.8-.1.1-.2.2h-.5c-.5-.1-.7-.6-.5-1.2 0-.4.2-.6.5-.7.1-.1.3-.2.6 0 .2 0 .3 0 .4.2v.3l.7-.5v-.1h-.2l-.1-.1-.1-.1c-.2-.2-.3-.3-.6-.3-.4-.1-.8 0-1 .1-.5.2-.7.6-.8 1-.3 1 .1 1.7 1 2l1-.1h.1zm2.4 1 .1-2.5v-.2h.2v-.1l-.9-.4v.3a2371.7 2371.7 0 0 0-2 2v.1l.7.4v-.3l.4-.4 1 .4-.1.5v.1H66l-.1.1.7.4.1-.1v-.2zm-.4-2.4-.2 1.2-.6-.3.8-.9zm3.2 4.3L71 62h.2v-.1l-.8-.6h-.1v.2l-.6.5-.9.6.2-.7.2-1v-.1h.2l-1-.7v.3L67 62.6h-.1v.1l.6.5V63l1.1-1.8-.3 1.5-.2.9h.1l.8-.5 1.3-1-.1.2A57.3 57.3 0 0 0 69 64h-.1l.7.5v-.2zm3.7 1.8h-.3l-1-.7a60.6 60.6 0 0 0 1.5-1.9l-.8-.6v.3l-1.4 2.2H71l1.6 1.2.1.2h.1l.4-.6zm2-1.4h.2l-.2.2-1.8 2h-.2l.7.5V67l.4-.3 1 .4a5 5 0 0 0-.2.6l-.1.1.7.4v-.3l.3-2.5v-.2s0-.1.2 0V65l-.9-.4zm.3.8-.2 1.2-.7-.4.9-.9zm3.2.4.1.2-.4 1.7c0 .3-.4.5-.7.4a.6.6 0 0 1-.4-.3c-.1 0-.1-.3 0-.4l.3-1.7.2-.1v-.1l-.9-.3v.1c.1.1.1.2 0 .2l-.3 1.7c-.2.8.1 1.2.9 1.4h1c.2-.2.4-.4.5-.8l.4-1.7h.2V66l-.8-.2v.1zm2.4.1h-1.3v3.1h1.1c.5 0 1 0 1.2-.3.3-.3.4-.7.4-1.2 0-1-.5-1.6-1.4-1.6zm.7 1.6c0 .8-.2 1-1 1h-.3v-2h.3c.8 0 1 .2 1 1zm3.8.2h-.2l-.1.2-1.3.3-.2-.7.7-.2h.3l-.2-.7-.2.1-.8.2-.1-.7a16.7 16.7 0 0 0 1.4-.2v-.7h-.2l-.2.1-2 .5v.1l.3.1.7 2.5-.2.2v.1l2.1-.5h.3l-.1-.7zm-35.8-5.7-2-1.8v-.1s-.1 0 0-.1v-.2l-1 .5v.1h.2v.2l.5 2.6v.3l.7-.4v-.1H48V63l-.1-.5.9-.5.4.4v.2l.8-.3v-.1l-.2-.1zm-1.4-.4-.6.3-.2-1.2.9.9z"/><path fill="#FFF" d="m41.9 35-1.1-.3.5 1.1-1 .8 1.2.2-.1 1.3 1-.9 1 .9-.1-1.3 1.2-.1-1-.9.6-1-1.3.3-.4-1.2-.5 1.2zm7.3 2-1.1-.4.6 1.1-1.1.7 1.2.3V40l1-.9.9.9-.1-1.3 1.3-.1-1-.8.5-1.2-1.2.4-.5-1.2-.5 1.2zM41 43.3l.5 1.2 1.2-.4-.6 1.1 1.1.8-1.3.1.1 1.3-1-.8-1 .8.1-1.2-1.2-.3 1.1-.7-.6-1.1 1.1.4zm8.7.5.3 1.1h1.2l-.9.7.3 1.1-.9-.6-.9.6.3-1.1-.9-.7h1.1zm-.5 8.8-1.2-.4.6 1.1-1 .7 1.2.3-.1 1.3 1-.9 1 .9-.2-1.3 1.3-.1-1-.9.6-1-1.3.3-.4-1.2-.5 1.2zM47 29.4c-.4 0-1 .5-1.2.7.8-.3 1-.1 1.2-.3v-.4zm.4.2c0 .5.3 1 .6 1.2l.1-.4c0-.3-.4-.6-.7-.8zm-.2.9c0 .1-.1.3 0 .4 0 .3.4.4.5.8.2-.4 0-1.1-.5-1.2zm-.2.8-.2.3c-.3.5.2.8.2 1.1.2-.2.4-.6.3-1 0-.2-.2-.3-.4-.4zm-1.2 0c-.2.1-.7.4-1 .4.2.2.7.1 1 .2.3 0 .5-.4.7-.7l-.7.1zm.5.9c-.6.1-.2.9-.8 1.4.4-.1.6-.1.8-.6.2-.2.2-.6 0-.8zm-5.1-1.5c-.1.5 0 .6.2 1-.1-.8.7-1.1.3-1.7-.2.1-.5.4-.5.7zm2.8 1.9c0 .2.7.2 1 0 0 0 .6-.2.6-.4-1.1-.2-.7.2-1.6.4zm1.3-1.6c.1.1.7 0 1-.2.1 0 .6-.3.6-.5-1.2 0-.7.3-1.6.7zm-3.8-1.5c-.1-.1-.6.1-.7.2-.3.1-.8.6-.6.7.7-.5.8 0 1.3-1zm1.6-2c.4 0 .6.3.9.4.3 0 .4-.1.4-.2 0-.4-.9-.4-1.3-.2zm1.9 1.7v-.9c0-.3-.2-.4-.3-.4-.3.3 0 1 .3 1.3zm-2-1.4c-.3-.1-.8.1-1 .3.4 0 .7.4 1.2.2l.3-.2a.8.8 0 0 0-.4-.3zm.7.8c0 .4.2 1 .3 1.1 0-.4.5-.7.2-1.2l-.2-.3-.3.4zm1.7 4.4H45c-.5 0-.5.6-1 .8.3 0 1 0 1.2-.3l.2-.5zm-3.7-4.5c-.5.2-.6.4-.7.7.5-.5 1.3 0 1.5-.6a.6.6 0 0 0-.8 0zm1.1 1c0 .5.2.6.5.8-.3-.7.4-1.2-.2-1.6-.2.1-.3.5-.3.8zm-.6-.1c0 .7 0 1.3.3 1.6.2-.5.2-.8 0-1.2l-.3-.4z"/></svg>
```

Here's a SVG that isn't as large in terms of file size (21 KB, as compared to 53 KB).